langchain4j-azure-open-ai/pom.xml

```diff
@@ -18,10 +18,16 @@
 
     <dependencies>
 
+        <dependency>
+            <groupId>dev.langchain4j</groupId>
+            <artifactId>langchain4j-core</artifactId>
```

Comment:
this should not be required: `langchain4j-open-ai` imports `langchain4j-core` transitively

Reply:
I'm guessing you mean the version of the dependency... Removed!